charts/operator-wandb/charts/mysql/templates/statefulset.yaml

```diff
@@ -49,12 +49,12 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: {{ include "wandb.mysql.passwordSecret" . }}
-                  key: MYSQL_ROOT_PASSWORD
+                  key: {{ if .Values.global.mysql.passwordSecret.key }}{{ .Values.global.mysql.passwordSecret.key }}{{ else }}MYSQL_ROOT_PASSWORD{{ end }}
```

Comment:
Is this right?  I imagine we would not want the `MYSQL_ROOT_PASSWORD` to be the same as the password that the application uses to access the database.  I think it should be a different key if we want to support someone setting it.